src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -465,4 +500,28 @@ public String getFilePath() {
 
         public boolean getSelected();
     }
+    
+    public static final class SelectByTag extends AbstractDescribableImpl<SelectByTag> {
+        private String testTag;
+        private static final String SELECT_BY_TAG = "SelectByTag";
+
+        @DataBoundConstructor
+        public SelectByTag(String testTag) {
+            this.testTag = Util.fixNull(testTag);
+        }
+
+        public String getTestTag() {
+            return this.testTag;
+        }
+
+        public void addTagToInputArgs(List<String> inputArgsList) {
+            // Concatenate all source folders to MATLAB cell array string.

```

Comment:
This comment doesn't look like it was updated after copy/paste

Reply:
:) Done